source/common/crypto/utility.h

```diff
@@ -47,6 +47,15 @@ class Utility {
   virtual std::vector<uint8_t> getSha256Hmac(const std::vector<uint8_t>& key,
                                              absl::string_view message) PURE;
 
+  /**
+   * Computes the SHA-256 HMAC for a given key and message.
+   * @param key the HMAC function key.
+   * @param message message data for the HMAC function.
```

Comment:
Meanwhile is there an option to re-use the existing function itself?

So that no code change is required on this common code as we as unit test update